prow/plank/controller.go

```diff
@@ -433,8 +434,11 @@ func (c *Controller) syncPendingJob(pj prowapi.ProwJob, pm map[string]corev1.Pod
 			return nil
 		}
 	}
-
-	pj.Status.URL = pjutil.JobURL(c.config().Plank, pj, c.log)
+	var err error
+	pj.Status.URL, err = pjutil.JobURL(c.config().Plank, pj, c.log)
+	if err != nil {
```

Comment:
The error we get here and below is terminal (as in: We know it will never succeed if it failed once. because we never update the DecorationConfig on a ProwJob), hence we should IMHO just log the error and proceed, otherwise jobs may get stuck in a given state

Reply:
Depends on outcome of: https://github.com/kubernetes/test-infra/pull/17183#discussion_r413253056

Reply:
Done